site/docs/components/carousel/examples.mdx

```diff
@@ -8,42 +8,33 @@ data:
   $ref: ./#/data
 ---
 
-## Default
+Carousel is a pre-defined pattern of Salt components that uses the [Embla](https://www.embla-carousel.com/) API.
 
-Use the default setting to allow users to navigate the carousel in single slide increments, moving either backward or forward through the sequence.
+You can customize the content within carousel slides according to your needs (see the [usage](./usage) documentation for guidance).
+Alternatively, the `CarouselCard` component offers a straightforward option for creating slide elements.
 
-<LivePreview componentName="carousel" exampleName="Default" />
+<LivePreview componentName="carousel" exampleName="Card" />
 
-## Multiple slides
-
-Use the multiple slides settings to allow users to navigate the carousel in single slide increments while displaying more than one slide at a time.
-
-To display multiple slides, pass multiple values (one per breakpoint) or a single value of the number of slides to show in `visibleSlides`.
-
-<LivePreview componentName="carousel" exampleName="MultipleSlides" />
+## Bordered
 
-## With title
+Bordered cards add a visual distinction to each slide, enhancing the overall appearance and making it easier for users to differentiate between individual slides.
 
-The carousel title is optional and should be placed in the top left corner of the carousel.
+`CarouselCard` can include a border by setting the `bordered` prop to `true`.
 
-<LivePreview componentName="carousel" exampleName="WithTitle" />
+<LivePreview componentName="carousel" exampleName="BorderedCard" />
```

Comment:
An observation, the progress text e.g. Slide 1 of 4, is really slow when updating. Is that normal?

Reply:
switched to a select, for things like the label and when waiting for the animation use a custom settle which is quicker